<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Connect_to_Server_Dialog</class>
 <widget class="QDialog" name="Connect_to_Server_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="IP_input">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>20</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="illustration">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter IP: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel_Button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Ok_Button">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>45</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_2">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>90</y>
     <width>45</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_3">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>90</y>
     <width>45</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>45</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_5">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>130</y>
     <width>45</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_6">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>130</y>
     <width>45</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>45</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_8">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>170</y>
     <width>45</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_9">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>170</y>
     <width>45</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_0">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>210</y>
     <width>45</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Delete">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>45</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_dot">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>210</y>
     <width>45</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
